<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmfDSWidget</class>
 <widget class="QWidget" name="EmfDSWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="le_title"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Titration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_titration"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_params">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <row>
          <property name="text">
           <string>emf0</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>emf0 flag</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>emf error</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>n</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>active</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>slope</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>slope flag</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>electrode0</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>380.0</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>0.02</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>1.0000</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>250</width>
           <height>35</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="table_data">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>65</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>V</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>emf</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
